plugins/woocommerce-admin/client/products/fields/attribute-field/attribute-field.scss

```diff
@@ -14,4 +14,52 @@
 	&__add-new {
 		margin: $gap-large 0 $gap-larger;
 	}
+	&__attribute-option-chip {
+		padding: 4px 8px;
+		gap: 2px;
+
+		background: $gray-100;
+		border-radius: 2px;
+	}
+
+	&__attribute-options {
```

Comment:
We might consider a min height.  As @joelclimbsthings noted in one of his sortable PRs, this can look odd if we drag the only item in the list:
<img width="576" alt="Screen Shot 2022-10-03 at 1 54 46 PM" src="https://user-images.githubusercontent.com/10561050/193680621-03f0cf2f-123d-4bab-8dd9-54bc8fdfa48c.png">

Reply:
I am not sure if that will help, cause the blue drop line doesn't show either when dragging a single item, this should maybe be something done on the Draggable component side?
Could we maybe allow us to disable drag if only one item is present?